libbeat/outputs/elasticsearch/client.go

```diff
@@ -333,6 +337,10 @@ func (client *Client) createEventBulkMeta(version common.Version, event *beat.Ev
 	return eslegclient.BulkIndexAction{Index: meta}, nil
 }
 
+func isRequireAliasSupported(version common.Version) bool {
+	return !version.LessThan(common.MustNewVersion("7.9.0"))
```

Comment:
The PR description mentions ES version 7.10. I'd expect either a `<= 7.9.0` check or a `< 7.10.0` check from that.